pydis_site/apps/redirect/redirects.yaml

```
@@ -87,9 +87,45 @@ resources_index_redirect:
   original_path: pages/resources/
   redirect_route: "resources:index"
 
-resources_resources_redirect:
-  original_path: pages/resources/<str:category>/
-  redirect_route: "resources:resources"
+resources_reading_redirect:
+  original_path: resources/reading/
+  redirect_route: "resources:index"
+  redirect_arguments: ["book"]
+
+resources_books_redirect:
+  original_path: resources/books/
+  redirect_route: "resources:index"
+  redirect_arguments: [ "book" ]
```

This should match the format of the other blocks.

```suggestion
  redirect_arguments: ["book"]
```